pkg/virtctl/guestfs/guestfs.go

```diff
@@ -363,15 +365,25 @@ func (c *guestfsCommand) setGIDLibguestfs() (*int64, error) {
 	return nil, nil
 }
 
-func (c *guestfsCommand) createLibguestfsPod(cmd string, args []string, isBlock bool) (*corev1.Pod, error) {
+func (c *guestfsCommand) createLibguestfsPod(client *K8sClient, ns, cmd string, args []string, isBlock bool) (*corev1.Pod, error) {
 	var resources corev1.ResourceRequirements
+	var tolerations []corev1.Toleration
+	var affinity corev1.Affinity
```

Comment:
```suggestion
	var affinity *corev1.Affinity
```

Should be okay to define it like this